src/mon/MonmapMonitor.cc

```diff
@@ -611,6 +611,17 @@ bool MonmapMonitor::prepare_command(MonOpRequestRef op)
       err = -EINVAL;
       goto reply_no_propose;
     }
+    if (monmap.stretch_mode_enabled) {
+      for (const auto &p : loc) {
+        if (!mon.osdmon()->osdmap.crush->name_exists(p.second)) {
+          ss << "location doesn't belong to any existing crush buckets!"
```

Comment:
very minor comment:
one long warning line? shouldn't it be broken down into multiple output lines?

Reply:
Several lines above also have one long warning line, IMHO the warning isn't so large that we need multiple lines.

Reply:
nvm I'm changing the output to be shorter, thank you for the review though.